german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/sicherer_Entfernung.md

```diff
@@ -0,0 +1,3 @@
+Sie fahren in die kleine Bucht ein und schauen sich die elektrisierte, wild tanzende Menge an. Am Rande der seltsamen Party entdecken Sie einen alten Herren, der mit verzueckter Miene zuhoert, sowie eine huebsche junge Frau, die neben ihm steht. 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.